### PR TITLE
Code together callout

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: code_together.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+### PennyLane: Code Together
+
+Join us on GitHub August 16th-27th. Be the first to solve an open issue with
+the "code together" label and win some awesome swag!
+
+### PennyLane: Code Together FAQ
+
+Q: What is PennyLane:Code Together? <br />
+A: PennyLane:Code Together is an online event where solving marked PennyLane
+issues rewards swag.
```
Comment (co9olguy) [inline code_together.md:10]:
```suggestion
A: PennyLane:Code Together is an online event where you can win swag for solving 
marked PennyLane.
```

File: code_together.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+the issues, feel free to comment on GitHub.
+
+Q: Can I create my own Code Together issue? <br />
+A: Yes! You can submit your own issue, just make sure to have the [Code
+Together] prefix in the issue title. We will then evaluate the issue for
+inclusion. If it receives the "code together" label, it will be part of the
+event!
+
+Q: Can a solution be created as a team? <br />
+A: Yes, teams comprising of more than one contributor can also submit solutions
```
Comment (co9olguy) [inline code_together.md:34]:
```suggestion
A: Yes, teams comprising more than one contributor can also submit solutions
```

File: code_together.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+to Code Together issues. It is worth noting, that we will be able to give out
```
Comment (co9olguy) [inline code_together.md:35]:
```suggestion
to Code Together issues. It is worth noting that we will be able to give out
```

**Review by co9olguy — APPROVED** [review]
:ok: thanks @antalszava

File: README.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+    <img width=50% src="https://pennylane.ai/blog/images/code_together.jpg">
+  </a>
+</p>
+
+<p align="center">
+  <strong>Announcing <a href="https://pennylane.ai/blog/2021/07/pennylane-code-together/">PennyLane: Code Together</a>!
+  Join us on GitHub August 16th-27th, see <a href="https://github.com/PennyLaneAI/pennylane/blob/master/code_together.md">event FAQ here</a></strong>.
+</p>
+
+<br>
```
Comment (josh146) [inline README.md:12]: Maybe might look slightly better with a horizontal line here, to separate the callout from the rest of the readme?

image


```suggestion
---
```

**Comment by antalszava** [comment]
[ch7880]

**Pull request merged**